src/schedule/schedule-week-view/schedule-week-view.styles.tsx

```diff
@@ -0,0 +1,243 @@
+`;
+export const Title = styled(Typography.BodyMD)<{
+    $isToday?: boolean;
+}>`
+    font-weight: 600;
+    ${({ $isToday }) =>
+        $isToday &&
+        css`
+            background: ${Colour["bg-hover"]};
+            color: ${Colour["text-primary"]};
+            border-radius: 50%;
+            border: ${Border["width-010"]} ${Border.solid}
+                ${Colour["bg-primary"]};
+            width: ${Spacing["spacing-40"]};
+            height: ${Spacing["spacing-40"]};
+            display: flex;
+            align-items: center;
+            justify-content: center;
+        `}
+`;
+
+export const Description = styled(Typography.BodyMD)`
+    font-weight: 600;
+`;
+
+export const BodyContainer = styled.div`
+    display: grid;
+    grid-template-columns: ${TIME_INDICATOR_WIDTH}px repeat(7, 1fr);
+    overflow-x: hidden;
+    overflow-y: auto;
+    flex: 1;
+    height: 100%;
+    border: ${Border["width-010"]} ${Border.solid} ${Colour["border"]};
+`;
+export const SlotGrid = styled.div`
+    grid-column-start: 2;
+    grid-column-end: 9;
+    display: grid;
+    grid-template-columns: repeat(7, 1fr);
+    min-width: max-content;
+`;
+
+export const Timeline = styled.div<{ $top: number }>`
+    position: absolute;
+    width: 100%;
+    height: 2px;
+    background: ${Colour["icon-primary"]};
+    top: ${({ $top }) => $top}px;
+    z-index: 2;
+
+    &::before {
+        content: "";
+        position: absolute;
+        left: -6px;
+        top: -6px;
+        width: 12px;
+        height: 12px;
+        border-radius: 50%;
+        background: ${Colour["icon-primary"]};
+        box-sizing: border-box;
+        z-index: 1;
+    }
+`;
+
+export const SlotColumn = styled.div`
+    display: flex;
+    position: relative;
+    flex-direction: column;
+    border-right: ${Border["width-010"]} ${Border.solid} ${Colour["border"]};
+    &:last-child {
+        border-right: none;
+    }
+`;
+
+export const SlotCell = styled.div<{
+    $startTime?: string;
+}>`
+    min-height: ${CELL_HEIGHT}px;
+    position: relative;
+    border-bottom: ${Border["width-010"]} solid ${Colour["border"]};
+    ${({ $startTime }) =>
+        $startTime?.endsWith(":00") &&
+        css`
+            border-bottom-style: dashed;
+        `}
+    cursor: pointer;
+`;
+
+export const SlotColumnOverlay = styled.div<{
+    $columnIndex: number;
+    $totalVisibleColumns: number;
+    $actualWidthPercentage?: number;
+    $leftPosition?: number;
+}>`
+    position: absolute;
+    top: 0;
+    left: ${({ $leftPosition }) => {
+        return `calc((100% - 14px) * ${$leftPosition} / 100)`;
+    }};
+    width: ${({ $actualWidthPercentage }) => {
+        return `calc((100% - 14px) * ${$actualWidthPercentage} / 100)`;
+    }};
+    height: 100%;
+    display: flex;
+    flex-direction: column;
+    pointer-events: none;
+    overflow: visible;
+    z-index: 1;
+    box-sizing: border-box;
+`;
+
+export const SlotContent = styled(Typography.BodyXS)<{
```

Comment:
separate the SlotContent component styles into `slot-content.styles.tsx`, same for TimeCell